lib/project_types/theme/commands/push.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+module Theme
+  module Commands
+    class Push < ShopifyCli::Command
+      options do |parser, flags|
+        parser.on('--remove') { flags['remove'] = true }
+        parser.on('--nodelete') { flags['nodelete'] = true }
+        parser.on('--allow-live') { flags['allow-live'] = true }
+      end
+
+      def call(args, _name)
+        if options.flags['remove']
+          remove = true
+          options.flags.delete('remove')
+          action = 'remov'
+        else
+          action = 'push'
+        end
+
+        flags = options.flags.map do |key, _value|
+          '--' + key
+        end
+
+        CLI::UI::Frame.open(@ctx.message('theme.checking_themekit')) do
+          Themekit.ensure_themekit_installed(@ctx)
+        end
```

Comment:
I wonder if this could be a prerequisite task that runs for all Themekit commands. Would that be possible?

Reply:
When the first couple commands were being written, `prerequisite_task` was broken; we're keeping this pattern for now, but we will change them all to a prerequisite task in bulk!